test-data/unit/check-inference.test

```diff
@@ -684,7 +684,9 @@ g('a')() # E: List[str] not callable
 # to backtrack later) and defaults to T = <nothing>. The result is an
 # awkward error message. Either a better error message, or simply accepting the
 # call, would be preferable here.
-g(['a']) # E: Argument 1 to "g" has incompatible type List[str]; expected List[<nothing>]
+g(['a']) # E: Argument 1 to "g" has incompatible type List[str]; expected List[<nothing>] \
+# N: "List" is invariant --- see http://mypy.readthedocs.io/en/stable/variance.html \
```

Comment:
This can now become http://mypy.readthedocs.io/en/stable/common_issues.rst#variance. (Or perhaps it's better to link to `latest`? `stable` won't have that label until we release 0.520.)